doc/classes/NetworkedMultiplayerCustom.xml

```diff
@@ -50,8 +51,9 @@
 			<argument index="1" name="buffer" type="PoolByteArray" />
 			<argument index="2" name="transfer_mode" type="int" />
 			<description>
-				Emitted when the local [MultiplayerAPI] generates a packet.
+				Emitted when the local [MultiplayerAPI] generates a packet (e.g. when calling [method Node.rpc]).
 				Your script should take this packet and send it to the requested peer over the network (which should call [method deliver_packet] with the data when it's received).
+				The [code]peer_id[/code] is the [code]target_peer[/code] set by [method NetworkedMultiplayerPeer.set_target_peer].
```

Comment:
I feel like `NetworkedMultiplayerPeer.set_target_peer()` is an implementation detail of `NetworkedMultiplayerPeer`, and this won't be very helpful to developers who are only familiar with the GDScript side of things.

I think it'd be better to just note that this is the target peer id (which could be 0 to denote sending to all peers), without mentioning `NetworkedMultiplayerPeer.set_target_peer()`

Reply:
You're probably right but since it's a custom class I felt unsure how much that responsibility was on me to implement if that makes sense.